backend/inspirehep/records/api/literature.py

```diff
@@ -237,11 +239,16 @@ def get_modified_references(self):
 
     def update_authors_signature_blocks_and_uuids(self):
         """Assigns a phonetic block and a uuid to each signature of a record.
+        Send the phonetic blocks to redis to be consumed by the disambiguation service.
```

Comment:
just relealized, this will send phonetic blocks even if the ``commit`` fails or never happens, it's better to move in on ``db.model.commited``